index.html

```diff
@@ -23,17 +24,164 @@
+
+
+    </section>
+
+<section class="middle-section">
+    <div class="tonic"><img alt="#" src ="./Work/Multi-post stories.png"> </div>
+    <div class ="header" id ="title"><h1>Multi-Post Stories</h1></div>
+    <ul class="details">
+        <li><div class ="canopy"><p>CANOPY</p></div></li>
+        <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+        <li><div class ="spec"><p>Back End Dev</p></div></li>
+        <li><div class ="counter"><img alt="#" src ="./Icons/Counter.png"></div></li>
+        <li><div class ="spec"><p>2015</p></div></li>
+
+    </ul>
+    <div class="info"> <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+    <ul class="lang">
+        <li><div class="hcj"><p>html</p></div></li>
+        <li><div class="hcj"><p>css</p></div></li>
+        <li><div class="hcj"><p>javascript</p></div></li>
+    </ul>
+    <div class ="see"><button class ="button"type ="button"><p>See Project</p></button></div>
+
+
+</section>
+<section class="about-sec">
+    <div class="header"><h3>About Myself</h3></div>
+    <div class="info"><p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="connect"><p>LET'S CONNECT</p></div>
+    
+    <ul class ="icons">
+        <li class="cat"><a href="#"><img alt="#" src = "./Icons/cat.png"></a></li>
+        <li class="linkdin"><a href="#"><img alt="#" src = "./Icons/Linkedin icon.png"></a></li> 
+        <li class="p"><a href="#"><img alt="#" src = "./Icons/Peace.png"></a></li>
+        <li class="twitter"><a href="#"><img alt="#" src = "./Icons/Twitter.png"></a></li>
+        <li class="m"><a href="#"><img alt="#" src = "./Icons/M.png"></a></li>
+              
+    </ul>
+    
+    <div class ="resume"><button class ="buttonr"type ="button"><p>Get my resume</p></button></div>
+    
+    <ul class ="dp">
+        
+        <li class="dropdown">Languages </li>
+        <li><img alt="#" src = "./Icons/down.png"></li>
+    </ul>
+    <ul class="line">
+        <li><img alt="#" src = "./Icons/Separator Bottom.png"></li>
+    </ul>
+
+    <ul class="ll">
+        <li><img alt="#" src="./Icons/js.png"></li>
+        <li class="dropdown">Javascript </li>
+
+    </ul>     
+    <ul class="ll">
+        <li><img alt="#" src="./Icons/html.png"></li>
+        <li class="dropdown">HTML </li>
+
+    </ul>  
+    <ul class="ll">
+        <li><img alt="#" src="./Icons/css.png"></li>
+        <li class="dropdown">CSS</li>
+
+    </ul>    
+    <ul class ="dp">
+        
+        <li class="dropdown">Framworks </li>
+        <li><img alt="#" src = "./Icons/down.png"></li>
+    </ul>
+    <ul class="line">
+        <li><img alt="#" src = "./Icons/Separator Bottom.png"></li>
```

Comment:
-  [ ] Kindly use a nested unordered list for your list of technologies. Your list of technologies should look like this;
```html
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```